mender-client.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
-MENDER_TENANT_TOKEN=${MENDER_TENANT_TOKEN}
+RUNNING_FILE=/tmp/mender-client-$$.run
+KEYS_DIR=${KEYS_DIR:-"./keys"}
+MENDER_SERVER_URL=${MENDER_SERVER_URL:-"https://127.0.0.1"} # "https://127.0.0.1"
```

Comment:
Why to change the default?

Reply:
Don't really remember already. Most likely I just pushed the one I used internally for dev purposes...